### PR TITLE
adding requirements for RALE

### LINKED ISSUE #2: Enable RALE Component
Using [RALE](http://devtools.web.roku.com/#rale-tool-link) should make UI development a lot easier. Currently not supported in the app and my little bit of playing with it cannot get the RALE component recognized.

### PR DESCRIPTION
**Pull request by mcrowson**
Adding the `TrackerTask.xml` file from the [RALE](http://devtools.web.roku.com/roku-advanced-layout-editor/) page so that we can use RALE to help layout scenes a bit better. 

**Changes**

Adding required XML component for RALE and referencing in the library scene that gets loaded. Instructions were found on the [Roku Forums](https://forums.roku.com/viewtopic.php?f=34&t=114969&p=593054&hilit=does+not+contain+a+RALE+component.&sid=0f092466da8a263f7eccda328903d6af#p593054)

**Issues**

Fixes #2

### CODE REVIEW

File: components/TrackerTask.xml [hunk 1]
```diff
@@ -0,0 +1,2085 @@
+<?rokuml version="1.0" encoding="utf-8" ?>
+<!--********** Copyright 2019 Roku Corp. All Rights Reserved. **********-->
```
Comment (bisby) [inline components/TrackerTask.xml:2]: Purely as an "I actually don't know" stance. Are there any issues with including the "Copyright Roku Corp" TaskTracker component into our code base directly?

File: components/config/scene.brs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 sub init()
     m.top.backgroundColor = "#000b35ff"
     m.top.backgroundURI = ""
-
+    m.tracker=m.top.createChild("TrackerTask")
```
Comment (bisby) [inline components/config/scene.brs:4]: Does BrightScript have a style guide, and should we be doing `x=y` or `x = y`? I've always done it with spaces (except in bash).

File: components/library/scene.xml [hunk 3]
```diff
@@ -12,4 +12,5 @@
     <field id="libraries" alias="LibrarySelect.libList" />
     <field id="librarySelected" alias="LibrarySelect.itemSelected" />
   </interface>
+  <script type="text/brightscript" uri="scene.brs"/>
```
Comment (bisby) [inline components/library/scene.xml]: I assume that relative pathing is ok, and we dont need the full `uri="pkg:/components/library/scene.xml"` ?

File: components/TrackerTask.xml (hunk 1 above)
Comment (mcrowson) [reply components/TrackerTask.xml:2]: Their instructions say to include it, so I think we're probably OK.

File: components/config/scene.brs (hunk 2 above)
Comment (mcrowson) [reply components/config/scene.brs:4]: I haven't seen anything in their docs and the linting in my VSCodium plugin doesn't seem to care one way or another. I'm all for consistency though if we want some internal conventions.

File: components/library/scene.xml (hunk 3 above)
Comment (mcrowson) [reply components/library/scene.xml]: Yup, relative paths are just fine. 

"BrightScript files can be imported using relative URIs. If the uri does not specify a complete pkg: file path, the uri will be interpreted as a file path relative to the XML file."

From [here](https://sdkdocs.roku.com/display/sdkdoc/script)

**Comment by bisby** [comment]
Does this make RALE universally usable or just for the Library screen?

**Review by bisby — APPROVED** [review]

**Comment by mcrowson** [comment]
It just has to find it on the first screen it loads. If the user is not authed then then it loadas it via the config screen and if they are authed it loads it from the library screen.

I'm no expert on this and have only monkied with it so far.

**Comment by bisby** [comment]
Cool. if we find that it's not loading somewhere we need it, we can always add it in later.

**Pull request merged**